tests/rptest/tests/full_disk_test.py

```diff
@@ -461,3 +461,77 @@ def test_basic_usage_report(self):
             self.redpanda.rolling_restart_nodes(self.redpanda.nodes)
 
             wait_until(lambda: self.check(0.05), timeout_sec=10, backoff_sec=5)
+
+
+class DiskStatsOverrideTest(RedpandaTest):
+    @cluster(num_nodes=3)
+    def test_disk_stat_total_overrides(self):
+        if not self.test_context.globals.get('use_xfs_partitions', False):
+            self.logger.warn(
+                "skipping test: looks like we are not using separate partitions for each node"
+            )
+            return
+
+        admin = Admin(self.redpanda)
+        # current size
+        orig_stat = admin.get_disk_stat("data", self.redpanda.nodes[0])
+        # set total size to 16 KB smaller
+        admin.set_disk_stat_override("data",
+                                     self.redpanda.nodes[0],
+                                     total_bytes=orig_stat["total_bytes"] -
+                                     2**14)
```

Comment:
nit: I see this 2**N around a lot -- is there a compelling reason for this vs some 16 * 1024? It seems more error prone to rely on **

Reply:
nah, it just looks l33t. switched to the easier to read version.